code/style.css

```diff
@@ -1,4 +1,212 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*GOOGLE FONT*/
+@import url('https://fonts.googleapis.com/css2?family=Staatliches&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap');
+
+.container {
+    width: 320px;
+    margin: 25px auto;
 }
+
+/**** HEADER ****/
+.header-landing-page {
+    background: #13141b;
+    height: 100px;
+    display:flex;
+    align-items: center;
+    padding: 0 16px;
+    margin: -8px;
+    font-family: 'Staatliches', cursive;
+    justify-content: space-between;    
+}
+
+.header-logo {
+    width: 180px;
+}
+.header-logo-container {
+    overflow:hidden;
+    width: 100px;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;   
+}
+
+.foot-step-right-side {
+    margin:5px;
+    display: none;
+}
+
+.site-title {
+    color:white;
+    font-size: 58px;
+    display: flex;
+}
+.navigation-container {
+    display: none;
+    font-size: 24px;    
+    background-color: #13141b;
+    color:rgb(9, 85, 32);     
+}
+
+.navigation-item {
+    margin-right: 20px;    
+}
+
+.navigation-item:last-child {
+    margin-right: 0px;
+}
+
+.nav-link:any-link {
+    color:rgb(9, 85, 32); 
+}
+/**** BIG NEWS ****/
+
+.big-news {
+    display: flex;
+    justify-content: space-around;
+    text-align: center;  
+    border: 2px solid black;
+    align-items: center;
+    padding:30px 0;
+    background: whitesmoke;
+    flex-flow: column wrap;
+    margin: 70px 20px 20px 20px;
+    width:90%;
+}  
+
+.big-news:hover {
+    box-shadow: 18px 0 18px grey;
+}
+
+.big-news-text{
+
+    width:200px;
+}
+
+.big-news-img {
+    padding:20px 20px 30px 20px;
+    border:1px solid black;
+    box-shadow: 5px 10px 10px grey;
+    border-radius: 1px;
+    filter: grayscale(100%); 
+}
+
+.big-news-img:hover {
+    padding:20px 20px 30px 20px;
+    border:1px solid black;
+    box-shadow: 18px 10px 18px black;
+    filter: grayscale(0%);
+}
```

Comment:
here you just need to specify what should change on the hover, so the only thing to put in this code block would be:
```
.big-news-img:hover {
    box-shadow: 18px 10px 18px black;
    filter: grayscale(0%);
}
```
the rest is inherited from the rules above.